packages/mongo-nav/src/mongo-select/Trigger.tsx

```diff
@@ -67,25 +72,44 @@ export function OrganizationTrigger({
   className,
   urls,
   isActive,
+  disabled = false,
   ...rest
 }: OrganizationTriggerProps) {
   return (
     <div className={cx(orgTriggerContainer, className)}>
-      <button {...rest} className={buttonContainer}>
-        <Icon size="small" glyph="Building" />
-        <span className={selectedStyle}>{placeholder}</span>
-        <Icon size="small" glyph="CaretDown" />
-      </button>
-      <a
-        href={urls.mongoSelect.orgSettings}
-        className={cx(anchorStyle, border)}
-        aria-label="settings"
+      <button
+        {...rest}
+        className={buttonContainer}
+        data-testid="org-trigger"
+        disabled={disabled}
       >
         <Icon
-          glyph={'Settings'}
-          fill={isActive ? uiColors.green.base : uiColors.gray.base}
+          className={cx({ [disabledStyle]: disabled })}
```

Comment:
Could we move this to a variable outside of the return statement?

As a neat trick, I believe this works:
```js
const disabledStyleClassName = cx({ [disabledStyle]: disabled })
return (
        <span className={cx(selectedStyle, disabledStyleClassName)}>
          {disabled ? 'All Organizations' : placeholder}
        </span>
)
```